core/components/molecules/stepper/__stories__/StepperWithAnimation.story.jsx

```diff
@@ -74,21 +74,25 @@ export const stepperWithAnimation = () => {
         }`}
         onAnimationEnd={handleAnimationEnd}
       >
-        <Card className="px-6 py-6">
+        <Card className="p-6">
           <Heading className="mb-6" size="s">
             Configure Initiative
           </Heading>
           <Text weight="strong">Population Filter</Text>
-          <div className="d-flex mt-5 mb-4">
-            <div className="mr-6" style={{ width: 'var(--spacing-8)' }}>
-              <Label withInput={true}>Region</Label>
-              <Dropdown options={options} />
-            </div>
-            <div style={{ width: 'var(--spacing-9)' }}>
-              <Label withInput={true}>Organization</Label>
-              <Dropdown options={options} />
-            </div>
-          </div>
+          <Row className="mt-5 mb-4">
+            <Column size={2}>
+              <div className="mr-6">
+                <Label withInput={true}>Region</Label>
+                <Dropdown options={options} />
+              </div>
+            </Column>
```

Comment:
Please add className `w-50` to Row and remove `size={2}` from the `Column` to preserve the style on the storybook and code sandbox both. 

<img width="260" alt="Screenshot 2023-01-10 at 1 24 57 AM" src="https://user-images.githubusercontent.com/46045493/211396161-ac1e3836-b565-4ce0-af4b-ee37575c01e5.png">

Reply:
w-50 is very different from the styling used here and in code sandbox as well it looks fine and varies according to the width of the browser window.